### PR TITLE
SUS-3108: Remove broken "Show contributions of new accounts only" option from Contributions

### PR DESCRIPTION
**Pull request by mszabo-wikia**
* practically not used
* implementation makes no sense in a shared DB setup
* Recent Changes can provide the same thing, but better

https://wikia-inc.atlassian.net/browse/SUS-3108

### CODE REVIEW

File: includes/specials/SpecialContributions.php [hunk 1]
```diff
@@ -560,7 +517,6 @@ function __construct( IContextSource $context, array $options ) {
 		}
 
 		$this->target = isset( $options['target'] ) ? $options['target'] : '';
-		$this->contribs = isset( $options['contribs'] ) ? $options['contribs'] : 'users';
 		$this->namespace = isset( $options['namespace'] ) ? $options['namespace'] : '';
 		$this->tagFilter = isset( $options['tagfilter'] ) ? $options['tagfilter'] : false;
```
Comment (WikiaTech) [inline includes/specials/SpecialContributions.php:519]: MINOR Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

File: includes/specials/SpecialContributions.php [hunk 2]
```diff
@@ -560,7 +517,6 @@ function __construct( IContextSource $context, array $options ) {
...
 		$this->nsInvert = isset( $options['nsInvert'] ) ? $options['nsInvert'] : false;
```
Comment (WikiaTech) [inline includes/specials/SpecialContributions.php:520]: MINOR Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

File: includes/specials/SpecialContributions.php [hunk 3]
```diff
@@ -86,15 +71,10 @@ public function execute( $par ) {
...
-			$out->addSubtitle( $this->contributionsSub( $userObj ) );
-			$out->setHTMLTitle( $this->msg( 'pagetitle', $this->msg( 'contributions-title', $target )->plain() ) );
-			$this->getSkin()->setRelevantUser( $userObj );
-		} else {
-			$out->addSubtitle( $this->msg( 'sp-contributions-newbies-sub' ) );
-			$out->setHTMLTitle( $this->msg( 'pagetitle', $this->msg( 'sp-contributions-newbies-title' )->plain() ) );
-		}
+		$target = $nt->getText();
+		$out->addSubtitle( $this->contributionsSub( $userObj ) );
+		$out->setHTMLTitle( $this->msg( 'pagetitle', $this->msg( 'contributions-title', $target )->plain() ) );
```
Comment (WikiaTech) [inline includes/specials/SpecialContributions.php:76]: MAJOR Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 24 issues
* ![MAJOR][MAJOR] 7 major
* ![MINOR][MINOR] 6 minor
* ![INFO][INFO] 11 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SpecialContributions.php#L38](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L38): The Cyclomatic Complexity of this function "execute" is 23 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L38](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L38): Reduce the number of returns of this function 5, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L38](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L38): This function "execute" has 154 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L163](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L163): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L197](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L197): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L334](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L334): This function "getForm" has 164 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)
1. ![MINOR][MINOR] [SpecialContributions.php#L494](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L494): Immediately return this expression instead of assigning it to the temporary variable "$form". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![MINOR][MINOR] [SpecialContributions.php#L523](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L523): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
1. ![MINOR][MINOR] [SpecialContributions.php#L528](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L528): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
1. ![MINOR][MINOR] [SpecialContributions.php#L529](https://github.com/Wikia/app/blob/addc5b736997e545766005aefc939946d3002ff2/includes/specials/SpecialContributions.php#L529): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by ghost — APPROVED** [review]

File: includes/specials/SpecialContributions.php [hunk 4]
```diff
@@ -422,21 +391,9 @@ protected function getForm() {
...
-			Xml::radioLabel(
-				$this->msg( 'sp-contributions-newbies' )->text(),
-				'contribs',
-				'newbie' ,
-				'newbie',
-				$this->opts['contribs'] == 'newbie',
-				array( 'class' => 'mw-input' )
-			) . '<br />' .
-			Xml::radioLabel(
+	Xml::label(
```
Comment (macbre) [inline includes/specials/SpecialContributions.php]: Indentation

**Review by macbre — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 24 issues
* ![MAJOR][MAJOR] 7 major
* ![MINOR][MINOR] 6 minor
* ![INFO][INFO] 11 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SpecialContributions.php#L38](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L38): The Cyclomatic Complexity of this function "execute" is 22 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L38](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L38): Reduce the number of returns of this function 5, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L38](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L38): This function "execute" has 152 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L163](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L163): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L195](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L195): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)
1. ![MAJOR][MAJOR] [SpecialContributions.php#L332](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L332): This function "getForm" has 164 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)
1. ![MINOR][MINOR] [SpecialContributions.php#L492](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L492): Immediately return this expression instead of assigning it to the temporary variable "$form". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![MINOR][MINOR] [SpecialContributions.php#L521](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L521): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
1. ![MINOR][MINOR] [SpecialContributions.php#L526](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L526): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)
1. ![MINOR][MINOR] [SpecialContributions.php#L527](https://github.com/Wikia/app/blob/ffc679f1b895b69c96bf4ca10b6089128306efdf/includes/specials/SpecialContributions.php#L527): Remove the literal "false" boolean value. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**